Source/UltraleapTrackingCore/UltraleapTracking.Build.cs

```diff
@@ -173,23 +171,26 @@ public bool LoadLeapLib(ReadOnlyTargetRules Target)
 
 				//Lib
 				PublicAdditionalLibraries.Add(Path.Combine(LibraryPath, PlatformString, "LeapC.lib"));
-
 				//System.Console.WriteLine("plugin using lib at " + Path.Combine(LibraryPath, PlatformString, "LeapC.lib"));
 
 				if (IsEnginePlugin())
 				{
 					PublicDelayLoadDLLs.Add("LeapC.dll");
-					RuntimeDependencies.Add(Path.Combine(BinariesPath, PlatformString, "LeapC.dll"));
-				}
+
+					// Copy third party DLLs to the BinariesPath 
+					RuntimeDependencies.Add(Path.Combine(BinariesPath, PlatformString, "LeapC.dll"), Path.Combine(LibraryPath, PlatformString, "LeapC.dll"));
+					RuntimeDependencies.Add(Path.Combine(BinariesPath, PlatformString, "LeapC.dll.manifest"), Path.Combine(LibraryPath, PlatformString, "LeapC.dll.manifest"));
```

Comment:
In this case we can't use .AddRange cause it can be only used to add modules, but in this case we need to add .dll and also copy them to the binaries folder